src/components/templates/project-list-wrapper.vue

```diff
@@ -14,8 +14,7 @@
 import MainList from '../organisms/main/main-list';
 import SectionTitle from '../atoms/common/section-title';
 
-// TODO: project도 나중에 vuex 에서 state로 관리해야 할까?
-import { projects } from '../static/data/projects.json';
+import { project } from '../static/data/index.js';
```

Comment:
```suggestion
import { project } from '../static/data';
```
마찬가지로 index.js 를 숨기는 것을 제안드립니다.